ci/scripts/test-concourse-runner.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/ash
+# shellcheck shell=dash
+set -euo pipefail
```

Comment:
You can't use set -u with the docker-helpers script :( It's not written in a way which works with erroring on unset variables